Gemfile

```diff
@@ -2,7 +2,8 @@ source 'https://rubygems.org'
 
 gemspec
 
-gem "aws-sdk", '~> 3'
+gem 'aws-sdk-ssm', '~> 1.55'
+gem 'aws-sdk-s3', '~> 1.73'
```

Comment:
I figured explicit requires might help us keep the deploy size leaner since `aws-sdk` has a billion dependencies

Reply:
I would recommend leaving the SDK gems in the layer.  Good to only install what you need.

Reply:
Yup this top-level Gemfile is what we use to run specs, etc

Reply:
I moved these into the `.gemspec` to make that clearer